packages/calcite-components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -81,17 +81,17 @@ export class ShellPanel implements ConditionalSlotComponent, LocalizedComponent,
    *
    * `"overlay"` displays at full height on top of center content, and
    *
-   * `"float"` does not display at full height with content separately detached from `calcite-action-bar` on top of center content.
+   * `"content-detached"` does not display at full height with content separately detached from `calcite-action-bar` on top of center content.
```

Comment:
fixme: This would be a breaking change since "float" no longer functions. If we need to add "content-detached" we should do so and not remove "float". "float" should be deprecated and removed at a major release.

Reply:
But it was deleted from the doc so that would not need to happen here.